Source/Blazorise/wwwroot/memoEdit.js

```diff
@@ -99,9 +99,10 @@ export function updateOptions(element, elementId, options) {
 
 function onInputChanged(e) {
     if (e && e.target) {
-        e.target.style.height = 'auto';
-        e.target.style.height = this.scrollHeight + 'px';
-        e.target.style.overflowY = 'hidden';
+        const textarea = e.target;
+        textarea.style.height = 'auto';
+        textarea.style.overflowY = 'hidden';
+        textarea.style.height = `${textarea.scrollHeight + 2}px`; // The +2 accounts for browser rendering inaccuracies with scrollHeight
```

Comment:
It's not an inaccuracy. It's caused by border, which is 1px from top and bottom.

```
 Style="border: 5px solid;"
```
will introduce the "bug" again.

![image](https://github.com/user-attachments/assets/b915c88e-e505-4143-9ef2-4db855b0d619)